### ISSUE TITLE
Where to find tensorboard output for visualization?

### ISSUE THREAD

**Issue by juanmed** [torch-points3d/torch-points3d#637]
Hi,

Let me first thank you and all contributors for developing such a great framework. After much reading and digging, I was able to create a custom loader for my data and train pointnet++ with it, but now I am trying to visualize the training curves with tensorboard and was unsuccessful. 

My train configuration is as below, with tensorboard log enabled (I assume) but I am unable to find a tensorboard output folder or file. I thought that running  ```tensorboard --logdir output/date_of_today/time_of_train/``` would allow me to visualize the training session but it does not work.

Any help on how to visualize the training curves (loss, miou, acc vs epoch) will be appreciated :)

Thanks

```yaml
training:
    epochs: 100
    num_workers: 6
    batch_size: 16
    shuffle: True
    cuda: 0
    precompute_multi_scale: False
    optim:
        base_lr: 0.001
        accumulated_gradient: 2
        grad_clip: -1
        optimizer:
            class: Adam
            params:
                lr: 0.001
        lr_scheduler:
            class: MultiStepLR
            params:
                milestones: [10, 25, 50, 75, 100]
                gamma: 0.1
        bn_scheduler:
            bn_policy: "step_decay"
            params:
                bn_momentum: 0.1
                bn_decay: 0.9
                decay_step : 1024
                bn_clip : 1e-2
    weight_name: "latest"
    enable_cudnn: True
    checkpoint_dir: ""
    resume: True

wandb:
    project: default
    log: False
    notes: ""
    name: ""
    public: True

tensorboard:
    log: True

```

**Comment by CCInc** [comment]
@juanmed I'm curious what commit of `master` your branch is based on?

**Comment by LucaRom** [comment]
For me I put the cd to `output/date_of_today/time_of_train/`. In this folder I have a tensorboard folder that is automatically created so I type : tensorboard `--logdir=tensorboard`. Then you need to open a web browser and connect to `http://localhost:6006` by default. For me it works only for runs that have run entirely and successfully.

**Comment by juanmed** [comment]
@CCInc  Hi, sorry for the late reply. My branch is based on commit ```5378a2ae7ef612433acedaa497073971ec66ff09```.

@LucaRom could you please share your ```torch-points3d/test/test_config/training_config.yaml``` file please? I have tried running tensorboard inside the ```time_of_train``` folder as you mentioned but still not working. My training runs are successful but still the tensorboard logs are not generated :(

**Comment by juanmed** [comment]
I was able to get the tensorboard output folder and get the training graphs by harcoding a ```True``` in the SegmentationTracker for my ```BaseDataset```, like:

```python
        return SegmentationTracker(self, wandb_log=wandb_log, use_tensorboard=True)
```

Either I am incorrectly configuring my .yaml file or not giving the correct parameters to Hydra (of which I have not idea how it works). Any comments?

**Comment by LucaRom** [comment]
I am using the training file for KPconv (`conf/training/kpconv.yaml`). It uses `tensorboard: log: True` as well. So I dont think it is the problem.

Are you explicitely calling the training file as an argument in the command line? `python train.py ... training=yourtrainingfilename`

**Comment by juanmed** [comment]
I was not explicitly calling the training file but after doing it the result is the same: not tensorboard folder unless I hardcode it (as mentioned above). As you said, ```tensorboard: log: True```  does not seem to be the problem. 

I tried the sample train command in the README file and added  ```training=default``` to specify a training configuration like this:
```bash
python train.py task=segmentation models=segmentation/pointnet2 model_name=pointnet2_charlesssg data=segmentation/shapenet-fixed training=default
```
but the result was the same, no tensorboard folder. There is something else going but I cannot figure out what is it.
